tracer/test/Datadog.Trace.TestHelpers.SharedSource/VerifyHelper.cs

```diff
@@ -280,6 +279,17 @@ public static VerifySettings AddSimpleScrubber(this VerifySettings settings, str
                        })
                   .OrderBy(x => x.Key)
                   .ToDictionary(x => x.Key, x => x.Value);
+
+            // In CI, we build the samples separately, so can end up with additional prefixes in the source file, dependent on the build agent
+            static string ScrubSourceFile(string path) =>
+                path switch
+                {
+                    _ when path.StartsWith(@"D:/a/1/s/") => path.Substring(9),
+                    _ when path.StartsWith(@"D:/a/_work/1/s/") => path.Substring(15),
```

Comment:
The new hosted pools use a _slightly_ different checkout path, which broke the file scrubbing for CI vis tests